{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>cryptography is a package designed to expose cryptographic primitives and recipes to Python developers. When certain sequences of <code>update()</code> calls with large values (multiple GBs) for symetric encryption or decryption occur, it's possible for an integer overflow to happen, leading to mishandling of buffers. This is patched in version 3.3.2 and newer.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/pyca/cryptography/security/advisories/GHSA-rhm9-p9w5-fwm7\">https://github.com/pyca/cryptography/security/advisories/GHSA-rhm9-p9w5-fwm7</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-36242\">https://nvd.nist.gov/vuln/detail/CVE-2020-36242</a></li>\n<li><a href=\"https://github.com/pyca/cryptography/issues/5615\">https://github.com/pyca/cryptography/issues/5615</a></li>\n<li><a href=\"https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst\">https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/L7RGQLK4J5ZQFRLKCHVVG6BKZTUQMG7E/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/L7RGQLK4J5ZQFRLKCHVVG6BKZTUQMG7E/</a></li>\n<li><a href=\"https://github.com/pyca/cryptography/commit/82b6ce28389f0a317bc55ba2091a74b346db7cae\">https://github.com/pyca/cryptography/commit/82b6ce28389f0a317bc55ba2091a74b346db7cae</a></li>\n<li><a href=\"https://github.com/pyca/cryptography/compare/3.3.1...3.3.2\">https://github.com/pyca/cryptography/compare/3.3.1...3.3.2</a></li>\n<li><a href=\"https://www.oracle.com/security-alerts/cpuapr2022.html\">https://www.oracle.com/security-alerts/cpuapr2022.html</a></li>\n<li><a href=\"https://www.oracle.com/security-alerts/cpujul2022.html\">https://www.oracle.com/security-alerts/cpujul2022.html</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-rhm9-p9w5-fwm7\">https://github.com/advisories/GHSA-rhm9-p9w5-fwm7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-rhm9-p9w5-fwm7",
            "title": "[cryptography] Symmetrically encrypting large values can lead to integer overflow",
            "date_modified": "2023-08-23T23:01:16.000Z",
            "date_published": "2021-02-10T01:32:27.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-rhm9-p9w5-fwm7"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Improper Limitation of a Pathname to a Restricted Directory",
            "date_modified": "2023-08-23T22:59:12.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Improper Limitation of a Pathname to a Restricted Directory",
            "date_modified": "2023-08-23T22:59:12.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Improper Limitation of a Pathname to a Restricted Directory",
            "date_modified": "2023-08-23T22:59:12.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>A <a href=\"https://cure53.de/fp170.pdf\">mutation XSS</a> affects users calling <code>bleach.clean</code> with all of:</p>\n<ul>\n<li><code>svg</code> or <code>math</code> in the allowed tags</li>\n<li><code>p</code> or <code>br</code> in allowed tags</li>\n<li><code>style</code>, <code>title</code>, <code>noscript</code>, <code>script</code>, <code>textarea</code>, <code>noframes</code>, <code>iframe</code>, or <code>xmp</code> in allowed tags</li>\n<li>the keyword argument <code>strip_comments=False</code></li>\n</ul>\n<p>Note: none of the above tags are in the default allowed tags and <code>strip_comments</code> defaults to <code>True</code>.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>Users are encouraged to upgrade to bleach v3.3.0 or greater.</p>\n<p>Note: bleach v3.3.0 introduces a breaking change to escape HTML comments by default.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<ul>\n<li><p>modify <code>bleach.clean</code> calls to at least one of:</p>\n<ul>\n<li>not allow the <code>style</code>, <code>title</code>, <code>noscript</code>, <code>script</code>, <code>textarea</code>, <code>noframes</code>, <code>iframe</code>, or <code>xmp</code> tag</li>\n<li>not allow <code>svg</code> or <code>math</code> tags</li>\n<li>not allow <code>p</code> or <code>br</code> tags</li>\n<li>set <code>strip_comments=True</code></li>\n</ul>\n</li>\n<li><p>A strong <a href=\"https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP\">Content-Security-Policy</a> without <code>unsafe-inline</code> and <code>unsafe-eval</code> <a href=\"https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src\"><code>script-src</code>s</a>) will also help mitigate the risk.</p>\n</li>\n</ul>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=1689399\">https://bugzilla.mozilla.org/show_bug.cgi?id=1689399</a></li>\n<li><a href=\"https://advisory.checkmarx.net/advisory/CX-2021-4303\">https://advisory.checkmarx.net/advisory/CX-2021-4303</a></li>\n<li><a href=\"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-23980\">https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-23980</a></li>\n<li><a href=\"https://cure53.de/fp170.pdf\">https://cure53.de/fp170.pdf</a></li>\n</ul>\n<h3 id=\"credits\">Credits</h3>\n<ul>\n<li>Reported by <a href=\"https://twitter.com/ynizry\">Yaniv Nizry</a> from the CxSCA AppSec group at Checkmarx</li>\n<li>Additional eject tags not mentioned in the original advisory and the CSP mitigation line being truncated in the revised advisory reported by <a href=\"https://twitter.com/SecurityMB\">Michał Bentkowski</a> at Securitum</li>\n</ul>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>If you have any questions or comments about this advisory:</p>\n<ul>\n<li>Open an issue at <a href=\"https://github.com/mozilla/bleach/issues\">https://github.com/mozilla/bleach/issues</a></li>\n<li>Email us at <a href=\"mailto:security@mozilla.org\">security@mozilla.org</a></li>\n</ul>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/mozilla/bleach/security/advisories/GHSA-vv2x-vrpj-qqpq\">https://github.com/mozilla/bleach/security/advisories/GHSA-vv2x-vrpj-qqpq</a></li>\n<li><a href=\"https://github.com/mozilla/bleach/commit/79b7a3c5e56a09d1d323a5006afa59b56162eb13\">https://github.com/mozilla/bleach/commit/79b7a3c5e56a09d1d323a5006afa59b56162eb13</a></li>\n<li><a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=1689399\">https://bugzilla.mozilla.org/show_bug.cgi?id=1689399</a></li>\n<li><a href=\"https://cure53.de/fp170.pdf\">https://cure53.de/fp170.pdf</a></li>\n<li><a href=\"https://github.com/mozilla/bleach/blob/79b7a3c5e56a09d1d323a5006afa59b56162eb13/CHANGES#L4\">https://github.com/mozilla/bleach/blob/79b7a3c5e56a09d1d323a5006afa59b56162eb13/CHANGES#L4</a></li>\n<li><a href=\"https://pypi.org/project/bleach/\">https://pypi.org/project/bleach/</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-23980\">https://nvd.nist.gov/vuln/detail/CVE-2021-23980</a></li>\n<li><a href=\"https://bugzilla.mozilla.org/show_bug.cgi?id=CVE-2021-23980\">https://bugzilla.mozilla.org/show_bug.cgi?id=CVE-2021-23980</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vv2x-vrpj-qqpq\">https://github.com/advisories/GHSA-vv2x-vrpj-qqpq</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vv2x-vrpj-qqpq",
            "title": "[bleach] Cross-site scripting in Bleach",
            "date_modified": "2023-08-23T22:57:37.000Z",
            "date_published": "2021-02-02T17:58:40.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vv2x-vrpj-qqpq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Pillow before 8.1.0, PcxDecode has a buffer over-read when decoding a crafted PCX file because the user-supplied stride value is trusted for buffer calculations.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-35653\">https://nvd.nist.gov/vuln/detail/CVE-2020-35653</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/6BYVI5G44MRIPERKYDQEL3S3YQCZTVHE/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/6BYVI5G44MRIPERKYDQEL3S3YQCZTVHE/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BF553AMNNNBW7SH4IM4MNE4M6GNZQ7YD/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BF553AMNNNBW7SH4IM4MNE4M6GNZQ7YD/</a></li>\n<li><a href=\"https://pillow.readthedocs.io/en/stable/releasenotes/index.html\">https://pillow.readthedocs.io/en/stable/releasenotes/index.html</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2021/07/msg00018.html\">https://lists.debian.org/debian-lts-announce/2021/07/msg00018.html</a></li>\n<li><a href=\"https://github.com/python-pillow/Pillow/commit/2f409261eb1228e166868f8f0b5da5cda52e55bf\">https://github.com/python-pillow/Pillow/commit/2f409261eb1228e166868f8f0b5da5cda52e55bf</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-f5g8-5qq7-938w\">https://github.com/advisories/GHSA-f5g8-5qq7-938w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-f5g8-5qq7-938w",
            "title": "[Pillow] Out-of-bounds Read",
            "date_modified": "2023-08-23T22:52:50.000Z",
            "date_published": "2021-03-18T19:55:41.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-f5g8-5qq7-938w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>clickhouse-driver before 0.1.5 allows a malicious clickhouse server to trigger a crash or execute arbitrary code (on a database client) via a crafted server response, due to a buffer overflow.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-26759\">https://nvd.nist.gov/vuln/detail/CVE-2020-26759</a></li>\n<li><a href=\"https://github.com/mymarilyn/clickhouse-driver/commit/3e990547e064b8fca916b23a0f7d6fe8c63c7f6b\">https://github.com/mymarilyn/clickhouse-driver/commit/3e990547e064b8fca916b23a0f7d6fe8c63c7f6b</a></li>\n<li><a href=\"https://github.com/mymarilyn/clickhouse-driver/commit/d708ed548e1d6f254ba81a21de8ba543a53b5598\">https://github.com/mymarilyn/clickhouse-driver/commit/d708ed548e1d6f254ba81a21de8ba543a53b5598</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vgv5-cxvh-vfxh\">https://github.com/advisories/GHSA-vgv5-cxvh-vfxh</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vgv5-cxvh-vfxh",
            "title": "[clickhouse-driver] Arbitrary code execution in clickhouse-driver",
            "date_modified": "2023-08-23T22:50:21.000Z",
            "date_published": "2021-04-07T20:50:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vgv5-cxvh-vfxh"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>petl before 1.68, in some configurations, allows resolution of entities in an XML document.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/petl-developers/petl/security/advisories/GHSA-f5gc-p5m3-v347\">https://github.com/petl-developers/petl/security/advisories/GHSA-f5gc-p5m3-v347</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-29128\">https://nvd.nist.gov/vuln/detail/CVE-2020-29128</a></li>\n<li><a href=\"https://github.com/petl-developers/petl/issues/526\">https://github.com/petl-developers/petl/issues/526</a></li>\n<li><a href=\"https://github.com/petl-developers/petl/pull/527\">https://github.com/petl-developers/petl/pull/527</a></li>\n<li><a href=\"https://github.com/petl-developers/petl/pull/527/commits/1b0a09f08c3cdfe2e69647bd02f97c1367a5b5f8\">https://github.com/petl-developers/petl/pull/527/commits/1b0a09f08c3cdfe2e69647bd02f97c1367a5b5f8</a></li>\n<li><a href=\"https://github.com/nvn1729/advisories/blob/master/cve-2020-29128.md\">https://github.com/nvn1729/advisories/blob/master/cve-2020-29128.md</a></li>\n<li><a href=\"https://github.com/petl-developers/petl/compare/v1.6.7...v1.6.8\">https://github.com/petl-developers/petl/compare/v1.6.7...v1.6.8</a></li>\n<li><a href=\"https://petl.readthedocs.io/en/stable/changes.html\">https://petl.readthedocs.io/en/stable/changes.html</a></li>\n<li><a href=\"https://github.com/petl-developers/petl/commit/07420ef8463cc387aea84e2d6241cf556574e2a5\">https://github.com/petl-developers/petl/commit/07420ef8463cc387aea84e2d6241cf556574e2a5</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-69q2-p9xp-739v\">https://github.com/advisories/GHSA-69q2-p9xp-739v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-69q2-p9xp-739v",
            "title": "[petl] XML Injection in petl",
            "date_modified": "2023-08-23T22:48:49.000Z",
            "date_published": "2021-04-20T16:32:08.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-69q2-p9xp-739v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>A malicious user can sign in as a user with any IndieAuth identifier. This is because the implementation does not verify that the final <code>\"me\"</code> URL value returned by the authorization server belongs to the same domain as the initial value entered by the user.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>Version 1.1 fixes this issue.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>There is no workaround. Upgrade to 1.1 immediately.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://indieauth.spec.indieweb.org/#differing-user-profile-urls-li-1\">Security Considerations: Differing User Profile URLs</a> in the IndieAuth specification.</li>\n</ul>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>If you have any questions or comments about this advisory:</p>\n<ul>\n<li>Open an issue in <a href=\"https://github.com/simonw/datasette-indieauth/issues\">simonw/datasette-indieauth</a></li>\n</ul>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/simonw/datasette-indieauth/security/advisories/GHSA-mjcr-rqjg-rhg3\">https://github.com/simonw/datasette-indieauth/security/advisories/GHSA-mjcr-rqjg-rhg3</a></li>\n<li><a href=\"https://pypi.org/project/datasette-indieauth/\">https://pypi.org/project/datasette-indieauth/</a></li>\n<li><a href=\"https://github.com/simonw/datasette-indieauth/commit/376c8804c6b0811852049229a24336fe5eb6a439\">https://github.com/simonw/datasette-indieauth/commit/376c8804c6b0811852049229a24336fe5eb6a439</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mjcr-rqjg-rhg3\">https://github.com/advisories/GHSA-mjcr-rqjg-rhg3</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mjcr-rqjg-rhg3",
            "title": "[datasette-indieauth] Implementation trusts the \"me\" field returned by the authorization server without verifying it",
            "date_modified": "2023-08-23T22:46:28.000Z",
            "date_published": "2020-11-24T21:21:04.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mjcr-rqjg-rhg3"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>The cache action in action/cache.py allows directory traversal through a crafted HTTP request. An attacker who can upload attachments to\nthe wiki can use this to achieve remote code execution.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>Users are strongly advised to upgrade to a patched version.</p>\n<p>MoinMoin Wiki 1.9.11 has the necessary fixes and also contains other important fixes.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>It is not advised to work around this, but to upgrade MoinMoin to a patched version.</p>\n<p>That said, a work around via disabling the <code>cache</code> or the <code>AttachFile</code> action might be possible.</p>\n<p>Also, it is of course helpful if you give <code>write</code> permissions (which include uploading attachments) only to trusted users.</p>\n<h3 id=\"credits\">Credits</h3>\n<p>This vulnerability was discovered by Michael Chapman.</p>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>If you have any questions or comments about this advisory, email me at <a href=\"mailto:twaldmann@thinkmo.de\">twaldmann@thinkmo.de</a>.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/moinwiki/moin-1.9/security/advisories/GHSA-52q8-877j-gghq\">https://github.com/moinwiki/moin-1.9/security/advisories/GHSA-52q8-877j-gghq</a></li>\n<li><a href=\"https://github.com/moinwiki/moin-1.9/commit/6b96a9060069302996b5af47fd4a388fc80172b7\">https://github.com/moinwiki/moin-1.9/commit/6b96a9060069302996b5af47fd4a388fc80172b7</a></li>\n<li><a href=\"https://pypi.org/project/moin/\">https://pypi.org/project/moin/</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-25074\">https://nvd.nist.gov/vuln/detail/CVE-2020-25074</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/11/msg00020.html\">https://lists.debian.org/debian-lts-announce/2020/11/msg00020.html</a></li>\n<li><a href=\"https://www.debian.org/security/2020/dsa-4787\">https://www.debian.org/security/2020/dsa-4787</a></li>\n<li><a href=\"http://moinmo.in/SecurityFixes\">http://moinmo.in/SecurityFixes</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-52q8-877j-gghq\">https://github.com/advisories/GHSA-52q8-877j-gghq</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-52q8-877j-gghq",
            "title": "[moin] remote code execution via cache action in MoinMoin",
            "date_modified": "2023-08-23T22:44:48.000Z",
            "date_published": "2020-11-11T15:54:51.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-52q8-877j-gghq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>lookatme versions prior to 2.3.0 automatically loaded the built-in \"terminal\" and \"file_loader\" extensions. Users that use lookatme to render untrusted markdown may have malicious shell commands automatically run on their system.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>Users should upgrade to lookatme versions 2.3.0 or above.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>The <code>lookatme/contrib/terminal.py</code> and <code>lookatme/contrib/file_loader.py</code> files may be manually deleted. Additionally, it is always recommended to be aware of what is being rendered with lookatme.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/pull/110\">https://github.com/d0c-s4vage/lookatme/pull/110</a></li>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/releases/tag/v2.3.0\">https://github.com/d0c-s4vage/lookatme/releases/tag/v2.3.0</a></li>\n</ul>\n<h3 id=\"for-more-information\">For more information</h3>\n<p>If you have any questions or comments about this advisory:</p>\n<ul>\n<li>Open an issue in <a href=\"https://github.com/d0c-s4vage/lookatme\">d0c-s4vage/lookatme</a></li>\n</ul>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/security/advisories/GHSA-c84h-w6cr-5v8q\">https://github.com/d0c-s4vage/lookatme/security/advisories/GHSA-c84h-w6cr-5v8q</a></li>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/pull/110\">https://github.com/d0c-s4vage/lookatme/pull/110</a></li>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/commit/72fe36b784b234548d49dae60b840c37f0eb8d84\">https://github.com/d0c-s4vage/lookatme/commit/72fe36b784b234548d49dae60b840c37f0eb8d84</a></li>\n<li><a href=\"https://github.com/d0c-s4vage/lookatme/releases/tag/v2.3.0\">https://github.com/d0c-s4vage/lookatme/releases/tag/v2.3.0</a></li>\n<li><a href=\"https://pypi.org/project/lookatme/#history\">https://pypi.org/project/lookatme/#history</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-15271\">https://nvd.nist.gov/vuln/detail/CVE-2020-15271</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-c84h-w6cr-5v8q\">https://github.com/advisories/GHSA-c84h-w6cr-5v8q</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-c84h-w6cr-5v8q",
            "title": "[lookatme] Markdown-supplied Shell Command Execution",
            "date_modified": "2023-08-23T22:43:38.000Z",
            "date_published": "2020-10-27T17:59:54.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-c84h-w6cr-5v8q"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h3 id=\"impact\">Impact</h3>\n<p>Metadadata signature verification, as used in <code>tuf.client.updater</code>, counted each of multiple signatures with identical authorized keyids  separately towards the threshold. Therefore, an attacker with access to a valid signing key could create multiple valid signatures in order to meet the minimum threshold of keys before the metadata was considered valid.</p>\n<p>The tuf maintainers would like to thank Erik MacLean of Analog Devices, Inc. for reporting this issue.</p>\n<h3 id=\"patches\">Patches</h3>\n<p>A <a href=\"https://github.com/theupdateframework/tuf/pull/974\">fix</a> is available in version <a href=\"https://github.com/theupdateframework/tuf/releases/tag/v0.12.2\">0.12.2</a> or newer.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>No workarounds are known for this issue.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-6174\">CVE-2020-6174</a></li>\n<li>Pull request resolving the issue <a href=\"https://github.com/theupdateframework/tuf/pull/974\">PR 974</a></li>\n</ul>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/theupdateframework/tuf/security/advisories/GHSA-pwqf-9h7j-7mv8\">https://github.com/theupdateframework/tuf/security/advisories/GHSA-pwqf-9h7j-7mv8</a></li>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-6174\">https://nvd.nist.gov/vuln/detail/CVE-2020-6174</a></li>\n<li><a href=\"https://github.com/theupdateframework/tuf/pull/974/commits/a0397c7c820ec1c30ebc793cc9469b61c8d3f50e\">https://github.com/theupdateframework/tuf/pull/974/commits/a0397c7c820ec1c30ebc793cc9469b61c8d3f50e</a></li>\n<li><a href=\"https://github.com/theupdateframework/tuf/pull/974\">https://github.com/theupdateframework/tuf/pull/974</a></li>\n<li><a href=\"https://github.com/theupdateframework/python-tuf/commit/2977188139d065ff3356c3cb4aec60c582b57e0e\">https://github.com/theupdateframework/python-tuf/commit/2977188139d065ff3356c3cb4aec60c582b57e0e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pwqf-9h7j-7mv8\">https://github.com/advisories/GHSA-pwqf-9h7j-7mv8</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pwqf-9h7j-7mv8",
            "title": "[tuf] Incorrect threshold signature computation in TUF",
            "date_modified": "2023-08-23T22:41:29.000Z",
            "date_published": "2020-08-21T16:25:26.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pwqf-9h7j-7mv8"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>This affects the package Gerapy from 0 and before 0.9.3. The input being passed to Popen, via the project_configure endpoint, isn’t being sanitized.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-7698\">https://nvd.nist.gov/vuln/detail/CVE-2020-7698</a></li>\n<li><a href=\"https://github.com/Gerapy/Gerapy/commit/e8446605eb2424717418eae199ec7aad573da2d2\">https://github.com/Gerapy/Gerapy/commit/e8446605eb2424717418eae199ec7aad573da2d2</a></li>\n<li><a href=\"https://snyk.io/vuln/SNYK-PYTHON-GERAPY-572470\">https://snyk.io/vuln/SNYK-PYTHON-GERAPY-572470</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-g57j-q48p-9vm2\">https://github.com/advisories/GHSA-g57j-q48p-9vm2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-g57j-q48p-9vm2",
            "title": "[gerapy] Command injection in Gerapy",
            "date_modified": "2023-08-23T22:39:58.000Z",
            "date_published": "2021-05-06T18:52:13.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-g57j-q48p-9vm2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was found in Apache Airflow versions 1.10.10 and below. When using CeleryExecutor, if an attacker can connect to the broker (Redis, RabbitMQ) directly, it is possible to inject commands, resulting in the celery worker running arbitrary commands.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-11981\">https://nvd.nist.gov/vuln/detail/CVE-2020-11981</a></li>\n<li><a href=\"https://lists.apache.org/thread.html/r7255cf0be3566f23a768e2a04b40fb09e52fcd1872695428ba9afe91%40%3Cusers.airflow.apache.org%3E\">https://lists.apache.org/thread.html/r7255cf0be3566f23a768e2a04b40fb09e52fcd1872695428ba9afe91%40%3Cusers.airflow.apache.org%3E</a></li>\n<li><a href=\"https://github.com/apache/airflow/commit/afa4b11fddfdbadb048f742cf66d5c21c675a5c8\">https://github.com/apache/airflow/commit/afa4b11fddfdbadb048f742cf66d5c21c675a5c8</a></li>\n<li><a href=\"https://issues.apache.org/jira/browse/AIRFLOW-6351\">https://issues.apache.org/jira/browse/AIRFLOW-6351</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-976r-qfjj-c24w\">https://github.com/advisories/GHSA-976r-qfjj-c24w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-976r-qfjj-c24w",
            "title": "[apache-airflow] Command injection via Celery broker in Apache Airflow",
            "date_modified": "2023-08-23T22:34:25.000Z",
            "date_published": "2020-07-27T16:57:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-976r-qfjj-c24w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An exploitable vulnerability exists in the YAML parsing functionality in the parse_yaml_query method in parser.py in MLAlchemy before 0.2.2. When processing YAML-Based queries for data, a YAML parser can execute arbitrary Python commands resulting in command execution because load is used where safe_load should have been used. An attacker can insert Python into loaded YAML to trigger this vulnerability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-16615\">https://nvd.nist.gov/vuln/detail/CVE-2017-16615</a></li>\n<li><a href=\"https://github.com/thanethomson/MLAlchemy/issues/1\">https://github.com/thanethomson/MLAlchemy/issues/1</a></li>\n<li><a href=\"https://github.com/thanethomson/MLAlchemy/commit/bc795757febdcce430d89f9d08f75c32d6989d3c\">https://github.com/thanethomson/MLAlchemy/commit/bc795757febdcce430d89f9d08f75c32d6989d3c</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-xpm8-98mx-h4c5\">https://github.com/advisories/GHSA-xpm8-98mx-h4c5</a></li>\n<li><a href=\"https://joel-malwarebenchmark.github.io/blog/2017/11/08/cve-2017-16615-critical-restful-web-applications-vulnerability/\">https://joel-malwarebenchmark.github.io/blog/2017/11/08/cve-2017-16615-critical-restful-web-applications-vulnerability/</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-xpm8-98mx-h4c5",
            "title": "[MLAlchemy] Unsafe deserialization in MLAlchemy",
            "date_modified": "2023-08-23T22:33:52.000Z",
            "date_published": "2018-07-13T16:01:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-xpm8-98mx-h4c5"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>urllib3 before version 1.23 does not remove the Authorization HTTP header when following a cross-origin redirect (i.e., a redirect that differs in host, port, or scheme). This can allow for credentials in the Authorization header to be exposed to unintended hosts or transmitted in cleartext.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-20060\">https://nvd.nist.gov/vuln/detail/CVE-2018-20060</a></li>\n<li><a href=\"https://github.com/urllib3/urllib3/issues/1316\">https://github.com/urllib3/urllib3/issues/1316</a></li>\n<li><a href=\"https://github.com/urllib3/urllib3/pull/1346\">https://github.com/urllib3/urllib3/pull/1346</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:2272\">https://access.redhat.com/errata/RHSA-2019:2272</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1649153\">https://bugzilla.redhat.com/show_bug.cgi?id=1649153</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-www2-v7xj-xrc6\">https://github.com/advisories/GHSA-www2-v7xj-xrc6</a></li>\n<li><a href=\"https://github.com/urllib3/urllib3/blob/master/CHANGES.rst\">https://github.com/urllib3/urllib3/blob/master/CHANGES.rst</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2021/06/msg00015.html\">https://lists.debian.org/debian-lts-announce/2021/06/msg00015.html</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/5SJERZEJDSUYQP7BNBXMBHRHGY26HRZD/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/5SJERZEJDSUYQP7BNBXMBHRHGY26HRZD/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BXLAXHM3Z6DUCXZ7ZXZ2EAYJXWDCZFCT/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BXLAXHM3Z6DUCXZ7ZXZ2EAYJXWDCZFCT/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XWP36YW3KSVLXDBY3QJKDYEPCIMN3VQZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XWP36YW3KSVLXDBY3QJKDYEPCIMN3VQZ/</a></li>\n<li><a href=\"https://usn.ubuntu.com/3990-1/\">https://usn.ubuntu.com/3990-1/</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00039.html\">http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00039.html</a></li>\n<li><a href=\"https://github.com/urllib3/urllib3/commit/560bd227b90f74417ffaedebf5f8d05a8ee4f532\">https://github.com/urllib3/urllib3/commit/560bd227b90f74417ffaedebf5f8d05a8ee4f532</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-www2-v7xj-xrc6",
            "title": "[urllib3] Exposure of Sensitive Information to an Unauthorized Actor in urllib3",
            "date_modified": "2023-08-23T22:32:49.000Z",
            "date_published": "2018-12-12T15:52:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-www2-v7xj-xrc6"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Ansible before versions 2.3.1.0 and 2.4.0.0 fails to properly mark lookup-plugin results as unsafe. If an attacker could control the results of lookup() calls, they could inject Unicode strings to be parsed by the jinja2 templating system, resulting in code execution. By default, the jinja2 templating language is now marked as 'unsafe' and is not evaluated.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-7481\">https://nvd.nist.gov/vuln/detail/CVE-2017-7481</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-w578-j992-554x\">https://github.com/advisories/GHSA-w578-j992-554x</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/a1886911fcf4b691130cfc70dfc5daa5e07c46a3\">https://github.com/ansible/ansible/commit/a1886911fcf4b691130cfc70dfc5daa5e07c46a3</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-w578-j992-554x",
            "title": "[ansible] Ansible fails to properly mark lookup-plugin results as unsafe",
            "date_modified": "2023-08-23T22:31:05.000Z",
            "date_published": "2018-09-06T03:28:50.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-w578-j992-554x"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Versions of Superset prior to 0.23 used an unsafe load method from the pickle library to deserialize data leading to possible remote code execution. Note Superset 0.23 was released prior to any Superset release under the Apache Software Foundation.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-8021\">https://nvd.nist.gov/vuln/detail/CVE-2018-8021</a></li>\n<li><a href=\"https://www.exploit-db.com/exploits/45933/\">https://www.exploit-db.com/exploits/45933/</a></li>\n<li><a href=\"https://github.com/apache/superset/pull/4243\">https://github.com/apache/superset/pull/4243</a></li>\n<li><a href=\"https://github.com/apache/superset/commit/2c72a7ae4fc0a8bac1f037a79efa90e1c5549710\">https://github.com/apache/superset/commit/2c72a7ae4fc0a8bac1f037a79efa90e1c5549710</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-vxp9-wv2f-wqmw\">https://github.com/advisories/GHSA-vxp9-wv2f-wqmw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-vxp9-wv2f-wqmw",
            "title": "[superset] Deserialization of Untrusted Data in superset",
            "date_modified": "2023-08-23T22:26:49.000Z",
            "date_published": "2018-11-09T17:40:56.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-vxp9-wv2f-wqmw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In PyYAML before 4.1, the yaml.load() API could execute arbitrary code. In other words, yaml.safe_load is not used.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-18342\">https://nvd.nist.gov/vuln/detail/CVE-2017-18342</a></li>\n<li><a href=\"https://github.com/marshmallow-code/apispec/issues/278\">https://github.com/marshmallow-code/apispec/issues/278</a></li>\n<li><a href=\"https://github.com/yaml/pyyaml/issues/193\">https://github.com/yaml/pyyaml/issues/193</a></li>\n<li><a href=\"https://github.com/yaml/pyyaml/pull/74\">https://github.com/yaml/pyyaml/pull/74</a></li>\n<li><a href=\"https://github.com/yaml/pyyaml/blob/master/CHANGES\">https://github.com/yaml/pyyaml/blob/master/CHANGES</a></li>\n<li><a href=\"https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation\">https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JEX7IPV5P2QJITAMA5Z63GQCZA5I6NVZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JEX7IPV5P2QJITAMA5Z63GQCZA5I6NVZ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/KSQQMRUQSXBSUXLCRD3TSZYQ7SEZRKCE/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/KSQQMRUQSXBSUXLCRD3TSZYQ7SEZRKCE/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/M6JCFGEIEOFMWWIXGHSELMKQDD4CV2BA/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/M6JCFGEIEOFMWWIXGHSELMKQDD4CV2BA/</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202003-45\">https://security.gentoo.org/glsa/202003-45</a></li>\n<li><a href=\"https://github.com/yaml/pyyaml/commit/7b68405c81db889f83c32846462b238ccae5be80\">https://github.com/yaml/pyyaml/commit/7b68405c81db889f83c32846462b238ccae5be80</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-rprw-h62v-c2w7\">https://github.com/advisories/GHSA-rprw-h62v-c2w7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-rprw-h62v-c2w7",
            "title": "[pyyaml] PyYAML insecurely deserializes YAML strings leading to arbitrary code execution",
            "date_modified": "2023-08-23T22:25:41.000Z",
            "date_published": "2019-01-04T17:45:26.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-rprw-h62v-c2w7"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>base/oi/doa.py in the Rope library in CPython (aka Python) allows remote attackers to execute arbitrary code by leveraging an unsafe call to pickle.load.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2014-3539\">https://nvd.nist.gov/vuln/detail/CVE-2014-3539</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1116485\">https://bugzilla.redhat.com/show_bug.cgi?id=1116485</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2015/02/07/1\">http://www.openwall.com/lists/oss-security/2015/02/07/1</a></li>\n<li><a href=\"https://github.com/python-rope/rope/commit/b01da7aab5cd02129941d2a900e6e5e3b5f7d4fb\">https://github.com/python-rope/rope/commit/b01da7aab5cd02129941d2a900e6e5e3b5f7d4fb</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-r38r-qp28-2m63\">https://github.com/advisories/GHSA-r38r-qp28-2m63</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-r38r-qp28-2m63",
            "title": "[rope] Code injection in rope",
            "date_modified": "2023-08-23T22:21:03.000Z",
            "date_published": "2018-07-26T16:08:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-r38r-qp28-2m63"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}